docs/farming-&-staking/staking/operators/deregister-operator.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Unregister an Operator
```

Comment:
To be consistent, let's call it de-register, as that's the name of the function that's called under the hood.